### ISSUE TITLE
Request: export additional entities for Wallet provider

### ISSUE THREAD

**Issue by Exzender** [tatumio/tatum-js#949]
It'd be convenient if we could import these entities from @tatumio/tatum

```
TatumConnector,
CreateErc1155NftCollection,
CreateFungibleToken,
CreateNftCollection,
```

**Comment by Smrecz** [comment]
Great catch, will export with next publish.

**Comment by Smrecz** [comment]
Released with 4.0.5

**Comment by Exzender** [comment]
Something is not right.
Installed package "@tatumio/tatum": "^4.0.5"

And receiving error while building project:

```
src/walletconnect.extension.ts:3:73 - error TS2305: Module '"@tatumio/tatum"' has no exported member 'TatumConnector'.
3      Constant, Utils, EVM_BASED_NETWORKS, Network, CreateFungibleToken, TatumConnector } from '@tatumio/tatum'
```

Here is import line
```
import { EvmRpc, TatumConfig, TatumSdkWalletProvider, ITatumSdkContainer, TxPayload,
     Constant, Utils, EVM_BASED_NETWORKS, Network, CreateFungibleToken, TatumConnector } from '@tatumio/tatum'
```

**Comment by Smrecz** [comment]
I don't think TatumConnector was ever exposed from the root

**Comment by Smrecz** [comment]
ohhh, I thought this is just about wallet provider types, didn't include TatumConnector !

**Comment by Smrecz** [comment]
@Exzender fixed in 4.0.6
